ggml-cuda.cu

```diff
@@ -7365,7 +7387,7 @@ static void ggml_cuda_mul_mat_mat_batched_cublas(const ggml_tensor * src0, const
                 dst->nb[2], dst->nb[3],
                 r2, r3);
         CUDA_CHECK(cudaGetLastError());
-
+        CUBLAS_CHECK(cublasSetStream(g_cublas_handles[id], main_stream));
```

Comment:
This is already done earlier in the function.

Reply:
Yeap. Removed.